package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "description": "training",
   "engines": {
-    "node": "6.11.1",
-    "npm": "2.14.2"
+    "node": "9.3.0",
```

Comment:
Heroku recommends this version? I would recommend using 8.9.4 which is the latests LTS version right now.